### PR TITLE
Generate Source code and compile

### PR DESCRIPTION
**Pull request by OscarTHZhang**
Fixed the compiler error and header error

### CODE REVIEW

File: generator/win_grpc_generator.cpp [hunk 1]
```diff
@@ -159,5 +178,97 @@ grpc::string GetHeaderServices(grpc_generator::File* file){
...
+  }
+  printer->Print(*vars, "return false;\n");
+  printer->Outdent();
+  printer->Print(*vars, "}\n");
+}
+
+void PrintSourceServerMethodSync(grpc_generator::Printer* printer,
+                            const grpc_generator::Method* method,
+                            std::map<grpc::string, grpc::string>* vars) {
+  (*vars)["Method"] = method->name();
```
Comment (youyuanwu) [inline generator/win_grpc_generator.cpp:214]: Rmove this function and only retain the variable in vars?

File: generator/win_grpc_generator.cpp [hunk 2]
```diff
@@ -159,5 +178,97 @@ grpc::string GetHeaderServices(grpc_generator::File* file){
...
+    // Scope the output stream so it closes and finalizes output to the string.
+    auto printer = file->CreatePrinter(&output);
+    std::map<grpc::string, grpc::string> vars;
+    // Package string is empty or ends with a dot. It is used to fully qualify
+    // method names.
+    vars["Package"] = file->package();
+    if (!file->package().empty()) {
+      vars["Package"].append(".");
+    }
+    vars["ns"] = "";
```
Comment (youyuanwu) [inline generator/win_grpc_generator.cpp]: You do you need this?

File: generator/win_grpc_generator.cpp [hunk 3]
```diff
@@ -159,5 +178,97 @@ grpc::string GetHeaderServices(grpc_generator::File* file){
   return output;
 }
 
+void PrintSourceServerRouterApplyMethodEntry(grpc_generator::Printer* printer,
+                                            const grpc_generator::Method* method,
+                                            std::map<grpc::string, grpc::string>* vars) {
+  if (method->NoStreaming()) {
+    printer->Print(*vars,
+      "if(url == L\"/$Package$$Service$/$Method$\") { \n"
+      "  auto op = std::bind(&Greeter::Service::SayHello, this, "
```
Comment (youyuanwu) [inline generator/win_grpc_generator.cpp]: Can you change the SayHello and Greeter to variables?

File: generator/win_grpc_generator.cpp (hunk 1 above)
Comment (OscarTHZhang) [reply generator/win_grpc_generator.cpp:214]: I had error after removing this function. Will retain this way

File: generator/win_grpc_generator.cpp [hunk 4]
```diff
@@ -159,5 +178,93 @@ grpc::string GetHeaderServices(grpc_generator::File* file){
...
+  }
+  printer->Print(*vars, "return false;\n");
+  printer->Outdent();
+  printer->Print(*vars, "}\n");
+}
+
+void PrintSourceServerMethodSync(grpc_generator::Printer* printer,
+                            const grpc_generator::Method* method,
+                            std::map<grpc::string, grpc::string>* vars) {
+  (*vars)["Method"] = method->name();
```
Comment (youyuanwu) [inline generator/win_grpc_generator.cpp:214]: Remove this in next PR.

**Review by youyuanwu — APPROVED** [review]

**Pull request merged**